test/train-sets/ref/cbe_cover_e_dec.stderr

```diff
@@ -1,23 +1,24 @@
 final_regressor = models/cover_e_dec.model
 predictions = cover_e_dec_train.pred
+warning: currently, mtr is only used for the first policy in cover, other policies use dr
```

Comment:
@olgavrou you worked recently on some change regarding this error message right? am I missing something?

Reply:
No that was the interactions warning message, this one looks like it has been around for a while

Reply:
it looks like it got added since the `cb_explore_adf_cover` reduction got enabled